src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -73,6 +75,7 @@ def setup(self,
         if 'TRY_TASK_ID' in os.environ and 'TRY_TASK_GROUP_ID' in os.environ:
             self.source = SOURCE_TRY
             self.try_task_id = os.environ['TRY_TASK_ID']
+            self.try_group_id = os.environ['TRY_TASK_GROUP_ID']
```

Comment:
Maybe settings is not the best place (or the best name, given that it is not just settings anymore), but it's OK for now.

Reply:
I can move that in the `Revision` if you prefer

Reply:
It's fine for now, as otherwise you'd have to move the rest too. We can clean things up after things settle.